### DISCUSSION TITLE
Add a radio extension where chatters can listen to and queue up music similar to plug.dj

### DISCUSSION THREAD

**Discussion by paragrips** [Chatterino/chatterino2#4905] [Ideas] [answered]
I believe plug.dj was really ahead of its time and it's a huge shame it got shut down. I believe Chatterino, being so widely used, could incorporate this into the client and introducing people who have never experienced plug.dj to that same experience. It seems quite niche, but I believe it does have some use in offline chats in particular.

**Comment by fourtf** [comment, marked as answer]
Thank you for your suggestion. While I can share your sentiment I am also hesitant to believe that it would be a good fit to be directly part of the Chatterino project. It seems to me that creating a web service and integration for this kind of feature may just be too much of a burden for the project to carry. Especially since it would need to be maintained over the years. I assume that this feature would also need to be accessible from a web browser to allow people who don't use Chatterino to access it. If we had support for a third party plugin to do this kind of thing then it might be possible but there would also extend the scope a lot since i believe most sites like plug.dj embedded YouTube into their site which isn't easily done in Chatterino.

**Comment by Felanbird** [comment]
what fourtf said, if `queup` has managed to not die for the 4th time by the time we fully add plugin support, someone could probably whip together a plugin for this (if possible)
